src/main/java/weiliang/bot/UI.java

```diff
@@ -0,0 +1,193 @@
+package weiliang.bot;
+
+import weiliang.bot.task.Deadline;
+import weiliang.bot.task.Event;
+import weiliang.bot.task.Task;
+import weiliang.bot.task.TaskList;
+
+/**
+ * Main ui class
+ */
+public class UI {
```

Comment:
Perhaps Ui or UserInterface may prevent abbreviations from being misunderstood or taken out of context.